src/app/examples/si-charts/interactive/interactive.html

```diff
@@ -67,55 +67,63 @@
       <div class="card-header">Options</div>
       <div class="card-body">
         <div class="mb-5 row">
-          <div class="form-check form-switch">
-            <input
-              class="form-check-input col-sm-2"
-              type="checkbox"
-              id="axisPointer"
-              role="switch"
-              [checked]="axisPointer"
-              (change)="toggleAxisPointer()"
-            />
-            <label class="form-check-label col-sm-8" for="axisPointer">Axis Pointer</label>
+          <div class="col">
```

Comment:
can you please also move those changes out of this PR as they are not yet needed